R/workhorse.R

```diff
@@ -1,18 +1,16 @@
 build_request <- function(path = NULL,
                           params = list(),
-                          method = "list",
-                          resource = "files",
+                          endpoint = "drive.files.list",
                           token = NULL,
```

Comment:
If the default value of `token` were `drive_token()`, that would remove a lot of repetition, yes?